### DISCUSSION TITLE
Some downloads are broken, what can I do?

### DISCUSSION THREAD

**Discussion by gyuseff** [yokoffing/Betterfox#297] [Q&A]
Hello, 
First of all, betterfox is great! I had been loving it :)

Though, I'm having issues downloading certain things. In my case I was not able to download some data from Facebook and from a goverment (Chilean) site. When I pressed the buttons, nothing happened. Do you know how I could fix that?

My workaround is use a non-betterfox profile when I want to download some stuff.

**Comment by yokoffing** [comment]
> When I pressed the buttons, nothing happened.

Thank you for reaching out. I noticed this behavior today when I was trying to do some work. Not sure if this is Firefox itself or a pref in Betterfox. I need to investigate.

General troubleshooting tips in the meantime: https://support.mozilla.org/en-US/kb/cant-download-or-save-files

**Reply by yokoffing** [reply]
This step seems to have worked for me on Google Drive, but I'm still making 100% sure on other sites: https://support.mozilla.org/en-US/kb/cant-download-or-save-files#w_reset-the-download-folder
